Extensions/editable_reblogs.js

```diff
@@ -341,6 +346,7 @@ XKit.extensions.editable_reblogs = new Object({
 
 	make_post: function(e) {
 		if (!this.reblog_tree_exists() || this.state != "success") {
+			XKit.interface.post_window.set_content_html(this.wrap_html_links(XKit.interface.post_window.get_content_html()));
```

Comment:
Good catch. :O There's at least one edge case that needs this to fire on submit rather than load, so I updated it to always remove the handler attribute and then re-add/fire it on form submit if the reblog tree hasn't been processed.